app/assets/js/rtMedia.js

```diff
@@ -206,18 +206,18 @@ function rtmedia_init_action_dropdown( parent ) {
 jQuery( 'document' ).ready( function( $ ) {
 
         // When Ajax completed attach media uploader to new activity, applay popup and attach media to comment uploader.
-        jQuery(document).ajaxComplete( function(event, xhr, settings) {
-            if ( bp_template_pack && bp_template_pack === 'nouveau' ) {  
-                var get_action = get_parameter( 'action', settings.data );
-                if ( ( 'activity_filter' === get_action ||  'post_update' === get_action || 'get_single_activity_content' === get_action || 'activity_get_older_updates' === get_action ) && typeof rtmedia_masonry_layout != 'undefined' && rtmedia_masonry_layout == 'true' && typeof rtmedia_masonry_layout_activity != 'undefined' && rtmedia_masonry_layout_activity == 'true' ) {
-                    setTimeout( function() {
-                        apply_rtMagnificPopup( '.rtmedia-list-media.rtm-gallery-list, .rtmedia-activity-container ul.rtmedia-list, #bp-media-list,.bp-media-sc-list, li.media.album_updated ul,ul.bp-media-list-media, li.activity-item div.activity-content div.activity-inner div.bp_media_content, .rtm-bbp-container, ul.rtm-comment-container' );
-                        rtmedia_activity_masonry();
-                        rtmedia_activity_stream_comment_media();
-                    } , 1000 );
-                }
-            }
-        } );
+		jQuery( document ).ajaxComplete( function( event, xhr, settings ) {
+			if ( bp_template_pack && 'legacy' !== bp_template_pack ) {
+				var get_action = get_parameter( 'action', settings.data );
+				if ( ('activity_filter' === get_action || 'post_update' === get_action || 'get_single_activity_content' === get_action || 'activity_get_older_updates' === get_action ) && 'undefined' !== typeof rtmedia_masonry_layout && 'true' === rtmedia_masonry_layout && 'undefined' !== typeof rtmedia_masonry_layout_activity && 'true' === rtmedia_masonry_layout_activity ) {
```

Comment:
@malavasita Have you tested this `if` condition?
Make sure `rtmedia_masonry_layout` and `rtmedia_masonry_layout_activity` contains string value `'true'` and not boolean `true`.

Reply:
Yes. It is coming from `rtMedia` options. and over there we are storing `string` only. @mangesh